Documentation/ExtensionArchitecture/DeclarationFile/Index.rst

```diff
@@ -6,20 +6,28 @@
 .. _extension-declaration:
 
 
-=================================
-Declaration File (ext_emconf.php)
-=================================
+=========================================
+Declaration file (:file:`ext_emconf.php`)
+=========================================
 
-*-- required*
+*-- required*  in legacy installations
 
-The :file:`ext_emconf.php` is the single most important file in an extension.
-Without it, the Extension Manager (EM) will not detect the extension, much less
-be able to install it. This file contains a declaration of what the extension
-is or does for the EM. The only thing included
+The :file:`ext_emconf.php` is used in legacy non-Composer installations to
+supply information about the extension to the :guilabel:`Extension Manager`. In
```

Comment:
```suggestion
supply information about the extension in the :guilabel:`Extensions` module. In
```

There is no *Extension Manager* label in the TYPO3 backend, only the *Extensions* module.